src/devtools/client/webconsole/components/Output/ConsoleOutput.tsx

```diff
@@ -232,11 +366,44 @@ function mapStateToProps(state: UIState) {
     visibleMessages: selectors.getVisibleMessages(state),
   };
 }
+
 const mapDispatchToProps = (dispatch: Dispatch) => ({
   dispatch,
-  openLink: actions.openLink,
 });
 
 const connector = connect(mapStateToProps, mapDispatchToProps);
 
 export default connector(ConsoleOutput);
+
+// TODO Is there a more central location for this utility method?
+async function getTimeForMessage(message: Message): Promise<number> {
```

Comment:
seems like it could reasonably be described as a timeline util? 🤷

Reply:
Yeah that was my _first_ instinct too, but the generic concept of points and time isn't really specific to the timeline. I dunno. Maybe that's still a reasonable enough place to put it though.

Reply:
I have an idea here.